### PR TITLE
New privacy policy

### PR DESCRIPTION
**Pull request by naterush**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context, and the specification if there is one. 

# Testing

Please provide a list of the ways you can "access" or use the functionality. Please try and be exhaustive here, and make sure that you test everything you list.

- [ ] I have tested this on real data that is reasonable and large
- [ ] If I changed the interaction with JupyterLab, I tested that it does not break other programs (like VS Code), and tested that it works "multiple times" in the same notebook.

# Documentation

Note if any new documentation needs to addressed or reviewed.

### CODE REVIEW

File: mitosheet/mitosheet/user/upgrade.py [hunk 1]
```diff
@@ -199,6 +199,18 @@ def upgrade_user_json_version_7_to_8(user_json_version_7: Dict[str, Any]) -> Dic
...
     return user_json_version_7
 
+def upgrade_user_json_version_8_to_9(user_json_version_8: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Adds the UJ_AI_PRIVACY_POLICY key, and sets it to False
+    """
+    # First, bump the version number
+    user_json_version_8[UJ_USER_JSON_VERSION] = 9
+
+    # Then, add the new enterprise key
```
Comment (aarondr77) [inline mitosheet/mitosheet/user/upgrade.py:209]: comment out of date

File: mitosheet/src/components/taskpanes/AITransformation/AITransformationTaskpane.tsx [hunk 2]
```diff
@@ -221,9 +251,31 @@ const AITransformationTaskpane = (props: AITransformationTaskpaneProps): JSX.Ele
...
+                        <p className="text-body-1">
+                            You can see Mito's <a className='text-underline' href='https://privacy.trymito.io/privacy-policy' target='_blank' rel="noreferrer">privacy policy here.</a>
+                        </p>
+                        <TextButton
+                            onClick={() => {
+                                void props.mitoAPI.updateAcceptAITransformationPrivacyPolicy();
+                            }}
+                            variant='dark'
+                        >
+                            Accept Privacy Policy for Mito AI
```
Comment (aarondr77) [inline mitosheet/src/components/taskpanes/AITransformation/AITransformationTaskpane.tsx:274]: When they accept, can we log it? Seems like something that we'd want a record of.

**Review by aarondr77 — COMMENTED** [review]
LGTM once tests pass. Before we do a wide-spread public release, I think we can improve the aesthetics of this privacy policy page to be more polished, transparent, and professional. Which might be particularly important for specific audiences. 

Relatedly, we'll probably want to do the work to give users more customization of what data they use. In my convo with a user on Friday, he thought that he wouldn't be able to use this on specific datasets because of the data collection. 

Something closer to the mockups you generated.

File: mitosheet/src/components/taskpanes/AITransformation/AITransformationTaskpane.tsx (hunk 2 above)
Comment (naterush) [reply mitosheet/src/components/taskpanes/AITransformation/AITransformationTaskpane.tsx:274]: We do log it -- all update events are logged by default.

**Comment by naterush** [comment]
Sounds great. Looking forward to iterating on all of these things with the AI feature going forward!

**Pull request merged**